app/Http/Controllers/MembersController.php

```diff
@@ -39,13 +46,13 @@ public function index()
 	}
 
 	# Member details page
-	public function show(Member $member)
+	public function show(Member $member, string $viewType = 'admin')
 	{
-		$viewType = 'admin';
-
 		return view('members.show', compact('member', 'viewType'));
 	}
 
+	// TODO creating and storing new members from here is never used -- we have a registration page... consider removing
```

Comment:
Sure, lets do it. I don't know if we have rights for "creating" participants and members, if there is, remember to check for it in the new controller.